shuup/admin/form_part.py

```diff
@@ -79,7 +81,14 @@ class SaveFormPartsMixin(object):
     request = None  # Placate "missing field" errors
     object = None  # --"--
 
+    @atomic()
     def save_form_parts(self, form):
+        # trigger signal for extra form validations
+        try:
+            view_form_valid.send(sender=type(self), view=self, form=form, request=self.request)
```

Comment:
Are you sure this is correct place for this? Should not this be inside the loop starting there at line 94?

Reply:
Do you think we should call this for every form inside the formgroup? IMO this should happen with the parent form.